dependencies/vaticle/repositories.bzl

```diff
@@ -32,11 +32,16 @@ def vaticle_typeql():
     )
 
 def vaticle_typedb_protocol():
+#    git_repository(
+#        name = "vaticle_typedb_protocol",
+#        remote = "https://github.com/vaticle/typedb-protocol",
+#        # NOTE: the sync-marker is also used for workspace status by Bazel!
+#        tag = "2.28.0",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typedb_protocol
+#    )
     git_repository(
         name = "vaticle_typedb_protocol",
-        remote = "https://github.com/vaticle/typedb-protocol",
-        # NOTE: the sync-marker is also used for workspace status by Bazel!
-        tag = "2.28.0",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @vaticle_typedb_protocol
+        remote = "https://github.com/farost/typedb-protocol", # TODO: Change back to vaticle when the deps PR is merged!
```

Comment:
Can we reference a commit from `vaticle/typedb-protocol` here or do we have to release it? I don't feel like [this PR](https://github.com/vaticle/typedb-protocol/pull/202) is a new feature to be released...

Reply:
This can be a commit, yes. Keep the `NOTE` in the same place and let this be a commit dependency for now. Next release will have to start from protocol, but that's not a change that needs a release on its own.
Release validation should catch this not being a tag dependency.

Reply:
sync dependencies updated the commit by itself, so I'll leave it as is. thanks!